### PR TITLE
Seemingly unrelated changes generated bug

### PR DESCRIPTION
**Pull request by 7i7o**
I saw a lot of projects showing the loading indicator in a loop when the browser didn't have the ethereum object or the user didn't connect any wallet

This bug was generated from code at the beggining of the UI section + adding the LoadingIndicator at the end of the project

Screen Shot 2021-10-27 at 15 49 37

### CODE REVIEW

**Comment by 7i7o** [comment]
Hi @adilanchian !

I already merged with buildspace:main (was only 2 commits behind that had no connection with this issue).
Before i started, i made sure of fetching the latest from your main. I even edited the whole thing in the github page so i was sure i didn't mess up. lol

Let me know if i am missing something

Eternally thankful for all the knowledge you guys are sharing!!!

File: NFT_Game/en/Section_4/Lesson_1_Finishing_Touches_UI.md [hunk 1]
```diff
@@ -113,6 +113,46 @@ const renderContent = () => {
...
+    } catch (error) {
+      console.log(error);
+    }
+    /*
+     * We release the state property after all the function logic
+     */
+    setIsLoading(false);
+};
+```
+
```
Comment (adilanchian) [inline NFT_Game/en/Section_4/Lesson_1_Finishing_Touches_UI.md:155]: So this is actually part of the `🔁 Adding loading indicators to App.js.` section! I don't think we need to make an entirely new section for this. 

Can you make the changes in that section instead?

File: NFT_Game/en/Section_3/Lesson_2_Building_Connect_Wallet_Button.md [hunk 2]
```diff
@@ -200,16 +200,16 @@ const App = () => {
...
-        setCurrentAccount(account);
-      } else {
-        console.log('No authorized account found');
+        if (accounts.length !== 0) {
+          const account = accounts[0];
+          console.log('Found an authorized account:', account);
+          setCurrentAccount(account);
+        } else {
+          console.log('No authorized account found');
+        }
```
Comment (adilanchian) [inline NFT_Game/en/Section_3/Lesson_2_Building_Connect_Wallet_Button.md:212]: also - I think you accidentally added a tab here - can you please revert?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Thanks for checking in on the latest for me :)

Left some comments! Let me know if you have any questions

File: NFT_Game/en/Section_4/Lesson_1_Finishing_Touches_UI.md (hunk 1 above)
Comment (7i7o) [reply NFT_Game/en/Section_4/Lesson_1_Finishing_Touches_UI.md:155]: Of course. As soon as I get home, I’ll merge it :)

File: NFT_Game/en/Section_3/Lesson_2_Building_Connect_Wallet_Button.md (hunk 2 above)
Comment (7i7o) [reply NFT_Game/en/Section_3/Lesson_2_Building_Connect_Wallet_Button.md:212]: This was actually on purpose, so it stays inside the “else” clause. 
Look in the previous JavaScript block, in the same lesson, that part of the code is inside the “else”.
I thought that you guys had unintentionally changed it 🤣

Comment (adilanchian) [reply NFT_Game/en/Section_3/Lesson_2_Building_Connect_Wallet_Button.md:212]: Haha AHH SHIT. Ya nice catch I totally didn't notice this 😅

ROGER. Can you do me a huge favor - could you make sure this code block is updated everywhere it is used in this project? I'll do a double check as well :)

Comment (7i7o) [reply NFT_Game/en/Section_3/Lesson_2_Building_Connect_Wallet_Button.md:212]: Sure. No problem. As soon as I get my hands on the computer I’ll check both 😊

File: NFT_Game/en/Section_4/Lesson_1_Finishing_Touches_UI.md (hunk 1 above)
Comment (7i7o) [reply NFT_Game/en/Section_4/Lesson_1_Finishing_Touches_UI.md]: Done, included this code block at the end of the the "🔁 Adding loading indicators to App.js." section.

File: NFT_Game/en/Section_3/Lesson_2_Building_Connect_Wallet_Button.md (hunk 2 above)
Comment (7i7o) [reply NFT_Game/en/Section_3/Lesson_2_Building_Connect_Wallet_Button.md:212]: Just checked it and this was the last time this function was shown (until that code block i had to add on the other comment thread).

**Comment by adilanchian** [comment]
EPIC. Thanks for working with me on this :). 

Nice work ✨

**Comment by adilanchian** [comment]
@all-contributors please add @7i7o for doc

**Pull request merged**